game/scripts/vscripts/libraries/timers.lua

```diff
@@ -117,9 +117,14 @@ function Timers:Think()
 
 		if v.endTime == nil then
 			v.endTime = now
+		elseif v.countTicks == true then
+			v.endTime = v.endTime - 1
+			if v.endTime < 0 then
```

Comment:
I think <= would look clearer. So, when requesting 1 tick I expect it to happen on the next loop, not in the second.